dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -3266,16 +3270,32 @@ def generate_issue_content(
     if latest:
         import requests
 
-        response = requests.get("https://pypi.org/pypi/apache-airflow/json")
-        response.raise_for_status()
-        latest_released_version = response.json()["info"]["version"]
-        previous = str(latest_released_version)
-        current = os.getenv("VERSION", "HEAD")
-        if current == "HEAD":
-            get_console().print(
-                "\n[warning]Environment variable VERSION not set, setting current release "
-                "version as 'HEAD'\n"
-            )
+        if not is_helm_chart:
```

Comment:
NIT: revert this check to avoid `not` and do not do the second check. Also you could extract what's inside the if statemantes to `get_latest_helm_chart()` and `get_latest_airflow` methods - and move them somwhere to the `utils` folder. That would make them reusable for other cases possibly.

```
if is_helm_chart:
....
else:
...
```

Reply:
Yeah that makes sense. Making the change